### ISSUE TITLE
Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT) 2024-07-02 > 2024-08-06

### ISSUE THREAD

**Issue by daniel-montalvo** [w3cping/privacy-request#139]
In the issue title above add the document name followed by the date of this request, then the date of your proposed deadline for comments.

- name of spec to be reviewed: Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)
- URL of spec: [This must be a dated version in the TR namespace, not an editor's draft.]: https://www.w3.org/TR/2024/DNOTE-wcag2ict-22-20240702/

- Current Rec/Note phase? Draft Note
- What and when is your next expected transition? Publish as a W3C Group Note after end of reviews 6 August.
- What has changed since any previous review? Updates since the 15 August 2023 draft include:
  * Added guidance for WCAG 2.2 success criteria
  * Addressed comments from the previous draft
  * Completed the Success Criteria Problematic for Closed Functionality section
- Please point to the results of your self-review (see https://w3ctag.github.io/security-questionnaire/ , https://w3c.github.io/fingerprinting-guidance/, https://tools.ietf.org/html/rfc6973):  NA -- This Note does not create/propose new technologies/accessibility features. Instead, it provides guidance on how WCAG could be applied to other non-Web ICT.
- Where and how to file issues arising? https://www.github.com/w3c/wcag2ict/issues/
- Pointer to any explainer for the spec? https://www.w3.org/WAI/standards-guidelines/wcag/non-web-ict/

Other comments:

**Comment by sandandsnow** [comment]
Hi @daniel-montalvo, thank you for requesting a review. I see that there is work on WCAG 3. Will this Group Note be updated when that work is complete to reflect guidance on applying WCAG 3 to Non-Web Information and Communications Technologies?

**Comment by daniel-montalvo** [comment]
@sandandsnow 
Thanks for your question.

WCAG3 is still years away. For more details, see [WCAG3 introduction](https://www.w3.org/WAI/standards-guidelines/wcag/wcag3-intro/)

We plan to publish the current WCAG2ICT (covering WCAG 2.0, 2.1, and 2.2), as a Note as soon as comments from this review period are addressed.

**Comment by sandandsnow** [comment]
@daniel-montalvo thank you. We discussed this privacy request today in our PING call. In the privacy considerations, where you state "This Working Group Note does not introduce any new privacy considerations", we would like to suggest adding the following text: "However, when implementing the WCAG 2.2 guidelines and success criteria in the context of non Web information and communication technologies, information about users' accessibility needs or preferences may be exposed and should be protected from disclosure and misuse in the non Web context. Further, when implementing the guidelines and success criteria, choose implementations that reduce the potential for fingerprinting or other identification and tracking of users, and only collect the data necessary to enable the accessibility features."

**Comment by sandandsnow** [comment]
Issue created in https://www.github.com/w3c/wcag2ict/issues/
Closing as privacy review complete
